edxval/utils.py

```diff
@@ -208,10 +211,12 @@ def get_transcript_format(transcript_content):
         transcript_content (str): Transcript file content.
     """
     try:
-        sjson_obj = json.loads(transcript_content)
+        sjson_obj = json.loads(transcript_content)  # pylint: disable=unused-variable
     except ValueError:
         # With error handling (set to 'ERROR_RAISE'), we will be getting
         # the exception if something went wrong in parsing the transcript.
+
+        # pylint: disable=len-as-condition
```

Comment:
I think the warning was just suggesting that you could more simply replace `if len(srt_subs) > 0` with `if srt_subs`.